openpype/modules/deadline/plugins/publish/submit_blender_deadline.py

```diff
@@ -180,3 +188,36 @@ def from_published_scene(self):
         the metadata and the rendered files are in the same location.
         """
         return super().from_published_scene(False)
+
+    @classmethod
+    def get_attribute_defs(cls):
+        defs = super(BlenderSubmitDeadline, cls).get_attribute_defs()
+        defs.extend([
+            BoolDef("use_published",
+                    default=cls.use_published,
+                    label="Use Published Scene"),
+
+            NumberDef("priority",
+                      minimum=1,
+                      maximum=250,
+                      decimals=0,
+                      default=cls.priority,
+                      label="Priority"),
+
+            NumberDef("chunkSize",
+                      minimum=1,
+                      maximum=50,
+                      decimals=0,
+                      default=cls.chunk_size,
+                      label="Frame Per Task"),
+
+            TextDef("group",
+                    default=cls.group,
+                    label="Group Name"),
+
+            TextDef("job_delay",
+                    default=cls.job_delay,
+                    label="Job Delay"),
```

Comment:
This could benefit from placeholder text: `dd:hh:mm:ss` just so it's clear what the user _should_ enter there if they were to ever clear it. I'd also say that adding that As a tooltip would be helpful as well.

The tooltip then also explain that the delay is added to the submission time.

Basically the tooltip could match the label in settings: `"Delay job (timecode dd:hh:mm:ss)"`

---

Out of curiosity, from where comes the request to "delay" the render?

Reply:
> 
> Out of curiosity, from where comes the request to "delay" the render?

I guess thats because of case when syncing data from cloud storage like Google / AWS taking some time to sync so u need some time for that in advance delaying the render job.

Reply:
But a short delay wouldn't be safe then though since you wouldn't _really_ know when the sync might be done? 

To wait for dependencies you should set up your job with `assetDependencies` so that Deadline only starts once those dependencies are found by deadline? Or? I believe that's what it was designed for at least.

Reply:
> But a short delay wouldn't be safe then though since you wouldn't _really_ know when the sync might be done?
> 
> To wait for dependencies you should set up your job with `assetDependencies` so that Deadline only starts once those dependencies are found by deadline? Or? I believe that's what it was designed for at least.

I do agree, but this being sort of simple workaround/way to do it too... similar being already done when speaking of DL integration for Celaction host.  This also comes from users needs aka they had been already using it before... and now not being possible/ option.

Reply:
Thanks - was just wondering about the request. Didn't know about the feature - thanks for the contribution @simonebarbieri ❤️

Reply:
Some DCCs do not provide the complete list of dependent files (even Maya with Yeti for example), so it is not reliable to use just this feature. In this specific case the client uses Google Drive and using the delay for job start is just bullet proof for them to allow the render scene (and other files) to sync correctly . . .